CMakeLists.txt

```diff
@@ -142,6 +137,122 @@ if(NOT ${NEBULA_GPERF_BIN_DIR} STREQUAL "")
     list(INSERT CMAKE_PROGRAM_PATH 0 ${NEBULA_GPERF_BIN_DIR})
 endif()
 
+# third-party
+if(NOT ${NEBULA_BZIP2_ROOT} STREQUAL "")
+    message(STATUS "Specified NEBULA_BZIP2_ROOT: " ${NEBULA_BZIP2_ROOT})
+    list(INSERT CMAKE_INCLUDE_PATH 0 ${NEBULA_BZIP2_ROOT}/include)
+    list(INSERT CMAKE_LIBRARY_PATH 0 ${NEBULA_BZIP2_ROOT}/lib)
+    list(INSERT CMAKE_PROGRAM_PATH 0 ${NEBULA_BZIP2_ROOT}/bin)
+endif()
```

Comment:
Newly added options, please add a comment at the beginning of the file

Reply:
Good advice